starter-code/main.js

```diff
@@ -53,16 +54,110 @@ mongoClient.connect(`mongodb://localhost:27017/crunchbase`, (error, db) => {
             })
             break;
 
-          // Code here next cases!
-        }
-      });
-    }
 
-    mainMenu()
-  }
-});
+        // 4.- List by name all companies founded in february of 2004.
+          case "4":
+
+            db.collection('companies')
+            .find({ "funding_rounds.funded_year": 2004, "funding_rounds.funded_month": 2}, { name: 1, _id: 0 })
+            .toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+            
+            // 5.- List by name all companies founded in the summer of 2004 (april to june) sorted by date.
+            
+          case "5":
+            db.collection('companies')
+            .find({ "funding_rounds.funded_year": 2004, "funding_rounds.funded_month": 4   }, { name: 1, _id: 0 })
```

Comment:
Aquí solo buscas los que fueran "fundados" en abril, el mes 4. 
Lo correcto es indicarle una horquilla:

`{ $and: [{founded_year: 2004}, { founded_month: {$gte:4} }, { founded_month: {$lte: 6}}] }`

$and para indicar más de una condición
$gte: mayor o igual que el mes 4
$lte: menor o igual que el mes 6

Con esas tres herramientas especificas que sean los meses del 4 al 6.